lib/src/lib/components/form-fields/autocomplete-field/autocomplete-field.component.html

```diff
@@ -26,7 +26,12 @@
     <mat-autocomplete #auto="matAutocomplete" [displayWith]="options.displayInputFn">
       @for (option of filteredOptions | async; track option) {
         <mat-option [value]="option?.value" [disabled]="option?.disabled">
+          <div class="option-container">
             <div [innerHTML]="option.label | translate"></div>
+          @if (option.color) {
+            <span class="color-circle" [style.background-color]="option.color"></span>
+          }
```

Comment:
My personal preference, but i'm not sure what the client wanted.
but i would put the status color circle in front of the value, instead of after

Reply:
or align them to the end of the div, like in the design in the cab ticket.